### PR TITLE
Clean up README

### PR DESCRIPTION
**Pull request by rgwood**
# Description

The README was getting very long and some places were out of date. I've done a pass to update out-of-date info and remove some info that the book covers better.

### CODE REVIEW

**Comment by fdncred** [comment]
thanks. it's been needing this pass through for a while. although, I think I'd vote to keep the installation section.

File: README.md [hunk 1]
```diff
@@ -155,72 +107,49 @@ For example, you can load a .toml file as structured data and explore it:
...
 
-### Configuration
-
-Nu has early support for configuring the shell. You can refer to the book for a list of [all supported variables](https://www.nushell.sh/book/configuration.html).
-
-To set one of these variables, you can use `config set`. For example:
-
-```shell
-> config set line_editor.edit_mode "vi"
-> config set path $nu.path
```
Comment (sholderbach) [inline README.md:204]: Wow yeah that is out of date

File: README.md [hunk 2]
```diff
@@ -16,78 +16,28 @@ A new type of shell.
...
-There is a [book](https://www.nushell.sh/book/) about Nu that is currently in progress.
-The book focuses on using Nu and its core concepts.
 
-If you're a developer who would like to contribute to Nu, we're also working on a [book for developers](https://www.nushell.sh/contributor-book/) to help you get started.
-There are also [good first issues](https://github.com/nushell/nushell/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22) to help you dive in.
 
-We also have an active [Discord](https://discord.gg/NtAbbGn) and [Twitter](https://twitter.com/nu_shell) if you'd like to come and chat with us.
 
-You can also find information on more specific topics in our [cookbook](https://www.nushell.sh/cookbook/).
+If you're a developer who would like to contribute to Nu, we're working on a [book for developers](https://www.nushell.sh/contributor-book/) to help you get started. There are also [good first issues](https://github.com/nushell/nushell/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22) to help you dive in.
```
Comment (sholderbach) [inline README.md:31]: I think the Contributor book is not yet updated and thus this leads to a stub.

**Review by sholderbach — COMMENTED** [review]
One suggestion maybe add a TOC at the top to quickly navigate to either pages in the same readme or to the appropriate page like I did it for reedline. This way, we can keep the full installation information (if up to date) there but can keep the easy and interesting to read stuff at the top.

**Comment by rgwood** [comment]
Thanks for the feedback; things got a bit busy over here but I’ll do another revision as soon as I have time.

**Comment by rgwood** [comment]
OK, did another pass. Notable changes:

- Updated `CONTRIBUTING.md`, it was very out of date
- Added some quick-install info back to the installation section
- Added a table of contents (the [Markdown All in One](https://marketplace.visualstudio.com/items?itemName=yzhang.markdown-all-in-one) extension is nice for this!)
- Removed mention of the contributor book since it's currently a work in progress

It occurrect to me that the Philosophy section could use a rewrite; it's currently a bit crowded, has lots of info that doesn't really fit under "Philosophy". But I'll leave that for another time.

File: README.md [hunk 3]
```diff
@@ -8,144 +7,112 @@
...
-To install Nu via the [Chocolatey](https://chocolatey.org) package manager:
+To quickly install Nu:
 
-```shell
-choco install nushell
+```bash
+# Linux and macOS
+brew install nushell
+# Windows
+winget install nu
```
Comment (fdncred) [inline README.md:47]: This conflicts what the documentation for winget https://www.nushell.sh/book/installation.html#package-managers

I'm not sure which is right.

File: README.md [hunk 4]
```diff
@@ -270,20 +210,15 @@ Nu is in heavy development and will naturally change as it matures and people us
 
 Please submit an issue or PR to be added to this list.
 
-###   Integrations
```
Comment (fdncred) [inline README.md:273]: Why are we deleting the integrations section?

Comment (rgwood) [reply README.md:273]: All the info's still there under the "Officially Supported By" header; I just thought it was unnecessary to have both "Officially Supported By" and "Integrations" headers.

File: README.md (hunk 3 above)
Comment (rgwood) [reply README.md]: `Nushell.Nushell` is the unambiguous ID for Nushell on winget, probably easiest to use that. Fixed.
image

Comment (rgwood) [reply README.md]: I'll update the website in a bit.

Comment (rgwood) [reply README.md]: On second thought.... `winget install nushell` works and it's easier to memorize. Will go with that.

**Pull request merged**